lib/student.ex

```diff
@@ -64,14 +112,21 @@ defmodule Student do
 
   def unsubscribe(_student, _project), do: {:error, :not_student}
 
+  def event_manager(%Student.Repr{pid: pid}) do
+    GenServer.call(pid, :event_manager)
+  end
+
+  # server side
+
   @impl true
   @spec init(String.t()) :: {:ok, %Student{}}
   def init(name) do
-    {:ok, %Student{name: name, projects: []}}
+    # XXX: handle error
+    {:ok, event_manager} = Student.Event.new()
```

Comment:
Looks like we're creating an Event here, not Event Manager.
We should consider renaming the function / creating submodule `Event.Manager`.